db/migrate/20231005035703_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false
+      t.string :encrypted_password, null: false
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
```

Comment:
- [ ] Nice job enabling the `confirmable` and `recoverable` in the user model, however, we do not need to create extra migration for the confirmation email. In the `db/migrate/20231005035703_add_devise_to_users.rb` file, devise has a way of enabling it for us by simply uncommenting `line 25 - 28` and then running the migration. I think because of the extra migration the app is not working as expected
![Screenshot from 2023-10-06 10-01-11](https://github.com/Elerqsousy/blog_ror/assets/46388113/87fc4a59-0685-4e1a-9ef3-6b3dd6ee5afb)

I was able to run the migration successfully after deleting the extra migration file, uncommenting the confirmable, and then running the migration. Let us kindly address this issue and make sure the app works as intended